Docs/database_naming_convention.md

```diff
@@ -0,0 +1,6 @@
+## Database Naming Convention
```

Comment:
What do you think about linking this from the main README?

I might even think about moving some of the Sequelize stuff from the main README into a separate doc in this folder and linking it from the main one.

Reply:
Went through and added a section to read me